src/modules/submissions.js

```diff
@@ -2017,6 +2032,31 @@
 		}
 	}
 
+	function checkIfUserIsBlocked( userName ) {
+		var deferred = $.Deferred();
+		AFCH.api.get( {
+			action: 'query',
+			list: 'blocks',
+			bkusers: userName
+		} ).then( function ( data ) {
+			var blocks = data.query.blocks;
+			var blockData = null;
+			var currentTime = new Date().toJSON();
```

Comment:
suggest using `toISOString` instead which has same output but makes the intent clear